project-management/src/main/java/life/qbic/projectmanagement/domain/model/sample/Sample.java

```diff
@@ -108,10 +108,17 @@ public SampleCode sampleCode() {
     return this.sampleCode;
   }
 
-  public SampleOrigin sampleOrigin() { return this.sampleOrigin; }
```

Comment:
please revert the code styling changes in this class

Reply:
no the change is correct according to our style guide :) 
```
void something() {
}
``` 
would be valid.